### ISSUE TITLE
Building with MBedTLS 3.6.0 LTS gives too old error.

### ISSUE THREAD

**Issue by ffreality** [nanomq/nanomq#1878]
**Describe the bug**
I am trying to build nanomq release `0.22.6` (I used `git submodule update --init --recursive`) with `MBedTLS`.
I have already an `MedTLS v3.6.0 LTS` built for my other projects such as `open62541` and it works as expected but when I try to set necessary paths on `cmake`, it gives `Mbed TLS too old (must be version 2 or newer)` error and `MbedTLS_DIR` path becomes empty.
It wants `cmake configuration file for mbedtls` and I showed `mbedtls source root` directory.

Should I use different path or nanomq doesn't support MBedTLS 3.X, yet ?

**Expected behavior**
Cmake should configure and generate solution files for Visual Studio.

**To Reproduce**
try to build nanomq with mbedtls 3.6.0 LTS

**Environment Details**
 - NanoMQ version = 0.22.6 Release
 - Operating system and version = Windows 11
 - Compiler and language used = Visual Studio 2022

**Comment by JaylinYu** [comment]
indeed, I see garrett has updated the mbdedtls ver, will follow it soon.
